src/ert/job_queue/driver.py

```diff
@@ -170,6 +240,7 @@ async def poll_statuses(self) -> None:
                 continue
             if tokens[0] not in self._lsfid_to_realstate:
                 # A LSF id we know nothing of, this should not happen.
+                logger.warning(f"Found unknown job id ({tokens[0]})")
```

Comment:
this is maybe an error

Reply:
Okay, I will raise a ValueError if this happens 💪